dll/win32/mstask/stubs.c

```diff
@@ -0,0 +1,133 @@
+
+#include "precomp.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(mstask);
+
+#define NET_API_STATUS DWORD
+typedef HANDLE ATSVC_HANDLE, SASEC_HANDLE;
+typedef struct _AT_INFO *LPAT_INFO;
```

Comment:
Where is `_AT_INFO` defined?

Reply:
psdk/lmat.h. I guess I can include that instead.

Reply:
atsvc_c.h was enough